code/week7/generic_class.h

```diff
@@ -0,0 +1,16 @@
+#include <iostream>
+
+template <class primitive_type> class Example {
+private:
+    primitive_type var;
+public:
+    Example(primitive_type var) : var(var) {
+        std::cout << "creating object with var = " << var << std::endl;
+    };
+    ~Example() {};
+
+    const primitive_type getVar() {
```

Comment:
I actually meant making the method const, not the return type 

ie: `primitive_type getVar() const`. I don't think it matters enough for us to care though.